src/sdk/model/company/cloud-tenant-bill.ts

```diff
@@ -66,6 +66,22 @@ export class CloudTenantBill {
     return this._json.full_month_contract_cost;
   }
 
+  /**
+   * Get has insider protection.
+   * @returns {boolean}
+   */
+  get hasInsiderProtection(): boolean {
+    return this._json.has_insider_protection;
+  }
+
+  /**
+   * Get insider protection price per gb.
+   * @returns {number}
+   */
+  get insiderProtectionPricePerGb(): number {
```

Comment:
@JoMas971 how do you know the currency?

Reply:
@cfsnyder there is a `currency_code` param that i'm using actually :-) 
Check `src/sdk/model/company/__json__/cloud-tenant-bill-json.ts` line 7

Reply:
That kinda sucks that it's on a completely separate model but not a problem on your side so LGTM